.ci/jobs/ecs-logging-php-mbp.yml

```diff
@@ -0,0 +1,5 @@
+---
+- job:
+    name: apm-agent-php/ecs-logging-php-mbp
```

Comment:
can we change the name to `ecs-logging-php/ecs-logging-php-mbp` ?

Reply:
Hey, `apm-agent-php` is the folder name and I put it explicitly as that's what it was done for the ecs-logging-java repo. The folders in the apm-ci are a kind of standardize with the agent name, that's the reason for this name, does it sound good to you? 

Thanks

Reply:
I see, yea sounds good 👍